index.html

```diff
@@ -91,36 +121,49 @@ <h1 class="project2-title">Profesional Art Printing Data</h1>
                     <button class="project2-button">See Project</button>   
             </div>
             <div class="project2">
-                <h1 class="project2-title">Profesional Art Printing Data</h1>
+                <div class="project2-3">
+                    <div class="desktop-hide2">
+                        <h1 class="project2-title">Profesional Art Printing Data</h1>
+                    </div>
+                        <h1 class="project2-title2">Data Dashboard Healthcare</h1>
                 <p class="project2-description">A daily selection of privately personalized reads; no accounts or
                     sign-ups required. has been the industry's standard</p>
                     <ul class="project2-tech">
                         <li><button>html</button></li>
                         <li><button>bootstrap</button></li>
                         <li><button>Ruby</button></li>
                     </ul>
+                </div>
                     <button class="project2-button">See Project</button>   
             </div>
             <div class="project2">
-                <h1 class="project2-title">Profesional Art Printing Data</h1>
+                <div class="project2-4">
+                    <div class="desktop-hide2">
+                        <h1 class="project2-title">Profesional Art Printing Data</h1>
+                    </div>
+                        <h1 class="project2-title3">Website Protfolio </h1>
                 <p class="project2-description">A daily selection of privately personalized reads; no accounts or
                     sign-ups required. has been the industry's standard</p>
                     <ul class="project2-tech">
                         <li><button>html</button></li>
                         <li><button>bootstrap</button></li>
                         <li><button>Ruby</button></li>
                     </ul>
+                </div>
                     <button class="project2-button">See Project</button>   
             </div>
         </div>          
     </section>
     <section id="about-section">
+        <div class="about-content">
         <div class="about-title">
             <h1>About me</h1>
         </div>
         <p class="about-description">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
         <button class="button-resume">Get My Resume</button>
+    </div>
         <div class="about-line"></div>
+        <div class="skill-section">
```

Comment:
- [ ] Same as here, I think you have introduced this broken style when fixing the 1st reviewer asked for changes, please fix this and ask for the other review, please

<img width="364" alt="1" src="https://user-images.githubusercontent.com/77843050/213576050-5d31c503-4982-4f26-8bf1-ab4b083f213e.png">